.github/workflows/icu_ci.yml

```diff
@@ -486,7 +486,7 @@ jobs:
 
   # Build Unicode update tools
   unicode-update-tools:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
The cache key is configured in this job. If the cache has worked for ubuntu-20.04, but the crossover to ubuntu-22.04 has caused problems, then maybe we can update the key so that caching is per-OS version. There are lots of [examples of that](https://github.com/actions/cache/blob/main/examples.md#rust---cargo).

So below where it says `key: bazel`, it would change to `key: ${{ runner.os }}-bazel`.